scanner/src/funTest/kotlin/storages/PostgresStorageTest.kt

```diff
@@ -43,7 +49,7 @@ class PostgresStorageTest : AbstractStorageTest() {
 
     init {
         "Embedded postgres works" {
-            EmbeddedPostgres.start().use { postgres ->
+            EmbeddedPostgres.builder().setPGStartupWait(PG_STARTUP_WAIT).start().use { postgres ->
```

Comment:
Unrelated to this change, but BTW, why to we start Postgres twice in this test?

Reply:
I think my plan was actually to delete this test in a later commit in the PR that introduced the PostgresStorage, but I forgot about it. I will review this after this PR is merged.